build.gradle

```diff
@@ -566,6 +566,23 @@ project(':iceberg-nessie') {
   }
 }
 
+project(':iceberg-dell') {
+  dependencies {
+    implementation project(':iceberg-core')
+    implementation project(':iceberg-common')
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation 'com.emc.ecs:object-client-bundle'
```

Comment:
For Aliyun and AWS, we both followed the pattern that service dependencies are expressed as `compileOnly`: https://github.com/apache/iceberg/blob/master/build.gradle#L280.

This allows us to bundle all the Iceberg dependencies to a runtime jar for Spark, Flink and Hive without any 3rd party dependencies, and if you need to run EMC ECS, then you add you service jar in addition to the Iceberg runtime jar in your own environment.

Reply:
Done.